content/guides/use-case/tensorflowjs.md

```diff
@@ -0,0 +1,584 @@
+   
+  static async setupCamera(cameraParam) {
+    if (!navigator.mediaDevices || !navigator.mediaDevices.getUserMedia) {
+      throw new Error(
+          'Browser API navigator.mediaDevices.getUserMedia not available');
+    }
+
+    const {targetFPS, sizeOption} = cameraParam;
+    const $size = VIDEO_SIZE[sizeOption];
+    const videoConfig = {
+      'audio': false,
+      'video': {
+        facingMode: 'user',
+        width: isMobile() ? VIDEO_SIZE['360 X 270'].width : $size.width,
+        height: isMobile() ? VIDEO_SIZE['360 X 270'].height : $size.height,
+        frameRate: {
+          ideal: targetFPS,
+        },
+      },
+    };
+
+    const stream = await navigator.mediaDevices.getUserMedia(videoConfig);
+
+    const camera = new Camera();
+    camera.video.srcObject = stream;
+
+    await new Promise((resolve) => {
+      camera.video.onloadedmetadata = () => {
+        resolve(video);
+      };
+    });
+
+    camera.video.play();
+
+    const videoWidth = camera.video.videoWidth;
+    const videoHeight = camera.video.videoHeight;
+    // Must set below two lines, otherwise video element doesn't show.
+    camera.video.width = videoWidth;
+    camera.video.height = videoHeight;
+
+    camera.canvas.width = videoWidth;
+    camera.canvas.height = videoHeight;
+    const canvasContainer = document.querySelector('.canvas-wrapper');
+    canvasContainer.style = `width: ${videoWidth}px; height: ${videoHeight}px`;
+    camera.ctx.translate(camera.video.videoWidth, 0);
+    camera.ctx.scale(-1, 1);
+
+    return camera;
+  }
+
+  drawCtx() {
+    this.ctx.drawImage(
+        this.video, 0, 0, this.video.videoWidth, this.video.videoHeight);
+  }
+
+  drawResults(faces, boundingBox, keypoints) {
+    drawResults(this.ctx, faces, boundingBox, keypoints);
+  }
+}
+```
+
+{{< /accordion >}}
+
+### The files in the src/shared/ directory
+
+The parameters and other shared files found in the `src/shared` directory are
+needed to run and access the camera, checks, and parameter values.
+
+## Explore the Docker assets
+
+In a Docker-based project, the Dockerfile and Compose file serve as foundational
+assets for building and orchestrating your application's environment. The
+Dockerfile defines the image that your container will run from, while the
+Compose file streamlines the deployment of multi-container applications,
+specifying how these containers interact with each other and with the host
+system. Explore the assets for this project in the following sections.
+
+### The Dockerfile
+
+A Dockerfile is a text file that instructs Docker how to create an image of your
+application's environment. An image contains everything you want and
+need when running application, such as files, packages, and tools.
+
+Open the `Dockerfile` in a code or text editor to explore its contents. This `Dockerfile` was generated by the [`docker init`](/reference/cli/docker/init/) command, with the following highlighted changes for this specific application.
```

Comment:
I guess we don't need to ask them to open the Dockerfile in a text editor since it's already on the page, in the accordion that follows

Are we saying that the highlighted lines are what's changed in this Dockerfile compared to what `docker init` generated? I couldn't really grok what the last sentence means

Reply:
The highlighting is inconsistent. I was trying to use it plus the tabs like highlight differences, but it's not intuitive. I think I'll remove the base docker init tabs, otherwise it'll start going too far into the weeds trying to explain/promote docker init.